docs/source/admin/quick_howto/index.rst

```diff
@@ -28,3 +28,4 @@ Traffic Control is a complicated system, and documenting it is not trivial. Some
 	ds_requests
 	steering
 	ciab
+	cachegroup_fallback
```

Comment:
Since the name of the file is `cachegroup_fallback_qht.rst` this won't include it. You could either change the filename or this line. It's up to you, but in my opinion it'd be best to change the filename; we already know it's a Quick How-to because it's in that section so I don't think the `_qht` adds any useful information.

Reply:
done